### PR TITLE
add types for localization keys

### PR DESCRIPTION
**Pull request by Val-M9**

### CODE REVIEW

File: packages/mobile/src/common/types/localization/loclization.d.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import 'react-i18next';
+import { resources } from '~/localization/i18n';
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/common/types/localization/loclization.d.ts:2]:
```suggestion
import { resources } from './i18n';
```

File: packages/mobile/src/localization/i18n.ts [hunk 2]
```diff
@@ -11,11 +16,8 @@ const i18 = () => {
     interpolation: {
       escapeValue: false,
     },
-    resources: {
-      en: { translation: en },
-      ua: { translation: ua },
-    },
+    resources: resources,
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/localization/i18n.ts:19]:
```suggestion
    resources,
```

File: packages/mobile/src/localization/localization.ts [hunk 3]
```diff
@@ -1 +1 @@
-export { i18 } from './i18n';
+export { i18, resources } from './i18n';
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/localization/localization.ts:1]:
```suggestion
export { i18 } from './i18n';
```

File: packages/mobile/src/localization/i18n.ts (hunk 2 above)
Comment (Val-M9) [reply packages/mobile/src/localization/i18n.ts]: fixed

File: packages/mobile/src/localization/localization.ts (hunk 3 above)
Comment (Val-M9) [reply packages/mobile/src/localization/localization.ts]: can't change, issues will hints won't work, and useTranslation get recursion depth issue

File: packages/mobile/src/common/types/localization/loclization.d.ts (hunk 1 above)
Comment (Val-M9) [reply packages/mobile/src/common/types/localization/loclization.d.ts:2]: can't change, issues will hints won't work, and useTranslation get recursion depth issue

**Review by AleksandrTermenzhy — APPROVED** [review]

**Review by seal323 — APPROVED** [review]

**Review by seal323 — APPROVED** [review]

**Pull request merged**